src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -361,7 +366,12 @@ export default Vue.extend({
 
     parseVideoData: function () {
       this.id = this.data.videoId
-      this.title = this.data.title
+
+      if (this.showDistractionFreeTitles) {
+        this.title = toDistractionFreeTitle(this.data.title)
```

Comment:
Modifying the title property will make `addToTitle` and `addToWatched` store the modified title instead of the original, which means that if a user disables the setting, the history and playlist view will still show the modified titles.

Ideally you want to add a new property that is only used for display purposes and modify that. That way the modified titles won't be stored in the databases.